### ISSUE TITLE
Auto Displace

### ISSUE THREAD

**Issue by rexmadden** [rexmadden/hansa#5]
Automatically place traders/merchants when there is only one possible available route.

**Comment by ctracey-nv** [comment]
Although unlikely, the rules state that when someone has been displaced, they may decline taking the additional tradesmen...  so technically there is always a choice for the opponent when someone takes the displace action.

**Comment by rexmadden** [comment]
You're no fun

**Comment by rexmadden** [comment]
It could be an option for the player. “Auto Displace when only one route possible” or something like that.

**Comment by ctracey-nv** [comment]
Yeah, it could work.  I’ll give it a try next time I’ve got some free time
